dataPrepII_PCC/makePCCRateFile.py

```diff
@@ -249,7 +252,7 @@ def doMakePCCRateFile(ConfigInfo):
     #    sys.exit(1)
 
     clusters = []
-    clusters = fillClusterCounts(pcc_tree, tree_scan_info, inData, PCC_BCID)
+    clusters = fillCounts(pcc_tree, tree_scan_info, inData, PCC_BCID, DataType)
```

Comment:
If the suggestion above concerning rates per layer is ok, add

PixelLayer as an argument